flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -139,6 +139,11 @@ under the License.
 			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
```

Comment:
Nit: Next commit?

Reply:
Doesn't the `flink-connector-kinesis` need to access the dependencies that have been moved to `flink-connector-aws-kinesis-data-streams` in the first commit? I moved the dependencies in the first commit and so added the dependency in the pom there too, otherwise it wouldn't compile.